packages/frontend/src/components/ElectionForm/Races/useEditRace.tsx

```diff
@@ -63,7 +62,18 @@ export const useEditRace = (race, race_index) => {
                 newErrors.raceDescription = 'Race title must be less than 1000 characters';
                 isValid = false;
             }
-        }
+            if (election.races.some(race => {
```

Comment:
Neat, I don't think I've used some() before but it looks handy

Reply:
I love all the array methods!